project/src/test/java/com/google/sps/servlets/AuthServletTest.java

```diff
@@ -1,80 +1,80 @@
-// Copyright 2021 Google LLC
```

Comment:
I think we should exclude changes to this file in this PR

Reply:
My bad, that was a mistake

Reply:
Fixed